ls1intum/Artemis
# Adapted quiz statistics bar labels to the behaviour of exam scores

### Checklist

### Motivation and Context


Requested by @krusche 

### Description

The code for placing the labels on the bar charts of the various quiz statistics was different from the one used in the exam scores modules. It was decided to adopt the exam scores code for the quiz statistics to create a uniform codebase.


### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create Quizzes, Participate in Quizzes
4. Check the various statistics available for quizzes for correct display(reachable under statistics in the exercise submenu)
![image](https://user-images.githubusercontent.com/29718932/91229441-23979200-e72a-11ea-85d1-1b32d240d860.png)

### Screenshots


![image](https://user-images.githubusercontent.com/29718932/91228988-702e9d80-e729-11ea-8249-5f2197e6ad20.png)
![image](https://user-images.githubusercontent.com/29718932/91229006-7886d880-e729-11ea-8314-074a9796e165.png)
